### PR TITLE
Tests2

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Supersedes #24.

File: src/lib.rs [hunk 1]
```diff
@@ -182,3 +182,21 @@ fn clockpoly(
...
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn internal() {
+        let args: Vec<String> = Vec::new();
+        let da = Params::from_iter(args);
```
Comment (dabreegster) [inline src/lib.rs]: Why not call this `params`? Matches the name of the type and doesn't take as much guesswork to realize it's "default arguments"

File: src/lib.rs [hunk 2]
```diff
@@ -182,3 +182,21 @@ fn clockpoly(
...
+        // println!("{}", da); Would be good to print and test args
```
Comment (dabreegster) [inline src/lib.rs]: If you're trying to do this, I think you want `{:?}` to show the debug representation

**Review by dabreegster — APPROVED** [review]

File: src/lib.rs (hunk 2 above)
Comment (Robinlovelace) [reply src/lib.rs]: That is a very useful trick. Still cannot see the output appearing in the output of `cargo test` but it now works, thanks!

File: src/lib.rs (hunk 1 above)
Comment (Robinlovelace) [reply src/lib.rs]: :+1:  done.

File: src/lib.rs (hunk 2 above)
Comment (dabreegster) [reply src/lib.rs]: The output from tests is hidden unless the test fails

**Pull request merged**